images/base/files/usr/local/bin/entrypoint

```diff
@@ -27,6 +27,11 @@ if grep -Eqv "0[[:space:]]+0[[:space:]]+4294967295" /proc/self/uid_map; then
   echo 'INFO: running in a user namespace (experimental)'
 fi
 
+grep_allow_nomatch() {
+  # grep exits 0 on match, 1 on no match, 2 on error
```

Comment:
> > EXIT STATUS
>        Normally the exit status is 0 if a line is selected, 1 if no lines were
>        selected, and 2 if an error occurred.  However, if the -q or --quiet or
>        --silent  is  used and a line is selected, the exit status is 0 even if
>        an error occurred.
> 

is it better to use --silent?

Reply:
`-q`, `--quiet`, and `--silent` are the same flag, and no that's not a suitable option:

>   -q, --quiet, --silent
>               Quiet; do not write  anything  to  standard  output.   Exit
>               immediately with zero status if any match is found, even if
>               an error was detected.  Also see the  -s  or  --no-messages
>               option.

Reply:
also we're not trying to swallow *errors*, we're remapping from:

- match: 0
- no-match: 1
- error: 2

to:

- match: 0
- no-match: 0
- error: non-zero

Reply:
ahh